src/test/java/lotto/domain/LottoTest.java

```diff
@@ -0,0 +1,67 @@
+package lotto.domain;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+
+class LottoTest {
+    private static final List<LottoNumber> LOTTO_NUMBERS = Arrays.asList(new LottoNumber(1), new LottoNumber(2), new LottoNumber(3), new LottoNumber(4), new LottoNumber(5), new LottoNumber(6));
+    @Test
+    void 로또_생성_성공() {
+        LottoNumberStrategy strategy = () -> IntStream.rangeClosed(1, 6)
+                .mapToObj(LottoNumber::new)
+                .collect(Collectors.toList());
+
+        Lotto lotto = new Lotto(LOTTO_NUMBERS);
+
+        assertThat(lotto).isNotNull();
+        assertThat(Lotto.create(strategy)).isEqualTo(new Lotto(LOTTO_NUMBERS));
+    }
+
+    @Test
+    void 로또_생성_실패() {
+        List<LottoNumber> invalidNumbers = Arrays.asList(
```

Comment:
@Displayname 을 통해 테스트이 설명을 조금 더 자세히 나타내보면 어떨까요? 😄 
```suggestion
    @Displayname("로또 번호가 6개 이하이면 로또를 생성할 수 없다.")
    @Test
    void 로또_생성_실패() {
        List<LottoNumber> invalidNumbers = Arrays.asList(
```